src/package/nuspec/Microsoft.NET.Test.Sdk.nuspec

```diff
@@ -9,12 +9,24 @@
     <requireLicenseAcceptance>true</requireLicenseAcceptance>
     <licenseUrl>http://www.microsoft.com/web/webpi/eula/net_library_eula_enu.htm</licenseUrl>
     <iconUrl>http://go.microsoft.com/fwlink/?LinkID=288859</iconUrl>
-    <description>The MSbuild targets and properties for building the .Net core test projects.</description>
+    <description>The MSbuild targets and properties for building .NET Core, .NET Full framework and Universal Windows test projects.</description>
     <copyright>© Microsoft Corporation. All rights reserved.</copyright>
     <projectUrl>https://github.com/microsoft/vstest/</projectUrl>
+    <tags>vstest visual-studio unittest testplatform mstest microsoft test</tags>
     <dependencies>
-      <dependency id="Microsoft.TestPlatform.TestHost" version="$Version$" />
-      <dependency id="Microsoft.CodeCoverage" version="1.0.3" />
+      <group targetFramework="uap10.0">
+        <dependency id="System.ComponentModel.Primitives" version="4.1.0" />
```

Comment:
Why did we choose to not provide TestHost in section? CodeCoverage is not provided because it's not supported in UWP yet.

A comment in here may help our future selves :)

Reply:
please do not hardcode any version, set them as min version.

Reply:
Done